docs/cookbook/creating_content_with_binary_attachments_via_rest_api.md

```diff
@@ -0,0 +1,147 @@
+# Creating content with binary attachments via REST API
+
+## Description
+
+This recipe illustrates how to create content via the REST API on the example of uploading an image file.
+
+## Solution
+
+The creation of the draft is done with a POST request on /api/ezp/v2/content/objects
+with a body containing the required and structured data.
```

Comment:
We should link to the documentation here: https://doc.ezplatform.com/en/1.11/guide/field_type_reference/#creating-content-data-property Reader will be able to find more details there.